pkg/publisher/publisher.go

```diff
@@ -86,14 +88,14 @@ func New(ctx context.Context) (Publisher, error) {
 	}
 	clusterID, err := ec2Client.GetClusterTag("CLUSTER_ID")
 	if err != nil || clusterID == "" {
-		glog.Errorf("Failed to obtain cluster-id, fetching name.  %v", err)
+		log.Errorf("Failed to obtain cluster-id, fetching name.  %v", err)
 		clusterID, err = ec2Client.GetClusterTag("Name")
 		if err != nil || clusterID == "" {
-			glog.Errorf("Failed to obtain cluster-id or name, defaulting to 'k8s-cluster'.  %v", err)
+			log.Errorf("Failed to obtain cluster-id or name, defaulting to 'k8s-cluster'.  %v", err)
```

Comment:
Use Infof here. The error is being "handled" and therefore you should simply log an informative message that it was handled instead of error.